scripts/cleanup_testfiles.sh

```diff
@@ -12,3 +12,4 @@ rm -f ./certeccrsa.der
 rm -f ./ecc-key.der
 rm -f ./ecc-key.pem
 rm -f ./ecc-public-key.der
+rm -f ./test-log-dump-to-file.txt
```

Comment:
Let's put this test file under the "./tests" directory, which would make this:

rm -f ./tests/test-log-dump-to-file.txt